JSON/testsuite/src/JSONTest.cpp

```diff
@@ -2369,7 +2369,23 @@ void JSONTest::testRemove()
 	assertTrue(nl[1] == "baz");
 
 }
-
+void JSONTest::testEnum()
+{
+	enum SAMPLE_ENUM
+	{
+		SOME_ENUM_VALUE = 42
+	};
+	
+	Poco::JSON::Object::Ptr obj = new Poco::JSON::Object();
+	
+	obj->set("taskType", SOME_ENUM_VALUE);
+	
+	Poco::Dynamic::Var var(obj);
+	std::string expected = "{\"taskType\":42}";
```

Comment:
What about converting the value back to enum with `operator T()` or `extract()`?